### PR TITLE
Run command test

### PR DESCRIPTION
**Pull request by devimc**
Implement tests for the run command

### CODE REVIEW

File: bundle.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	err = bundle.Save()
+	if err != nil {
+		return nil, err
+	}
+
+	return bundle, nil
+}
+
+// createRootfs creates a rootfs in the specific bundlePath
+func createRootfs(bundlePath string) error {
```
Comment (jodh-intel) [inline bundle.go]: Maybe return an error immediately if `bundlePath == ""`?

File: bundle.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+// Save to disk the Config
+func (b *Bundle) Save() error {
+	content, err := json.Marshal(b.Config)
+	if err != nil {
+		return fmt.Errorf("failed to marshal config: %s", err)
+	}
+
+	configFile := filepath.Join(b.Path, "config.json")
+	err = ioutil.WriteFile(configFile, content, 0664)
```
Comment (jodh-intel) [inline bundle.go]: Shouldn't that be `0644`? Related:

- What user do we expect this code to run as?
- Do we need to assert that?

Comment (devimc) [inline bundle.go]: yes ```0644``` is ok

> What user do we expect this code to run as?

I think only root is able to start containers

> Do we need to assert that?

I do not think so

File: bundle.go (hunk 1 above)
Comment (devimc) [inline bundle.go]: fixing
thanks

**Comment by chavafg** [comment]
lgtm

**Comment by devimc** [comment]
depends on https://github.com/clearcontainers/runtime/pull/209

**Pull request merged**